deployer.rb

```diff
@@ -41,26 +41,27 @@ def process_circle_ci(payload)
   end
 
   def process_pull_request(pull_request)
```

Comment:
You might be able to clean up the duplication here.

For example, if you changed the `hotfix?` method to take a `pull_request` object, like the `leankit_ticket_present?`, they you could have something like this:

```ruby
description = 
  if leankit_ticket_present?(pull_request)
    hotfix?(pull_request) ? "This is a hotfix!" :  slack_channel_topic
  else
    "None of the messages in the PR include LeanKit ID"

status = 
  leankit_ticket_present?(pull_request) || hotfix?(pull_request) 
    ? build_status 
    : "error",

client.create_status(
  repo_name(pull_request),
  pull_request['head']['sha'],
  status,
  opts.merge(description: description)
)
```

Reply:
And normally when I start creating variables in a method, I think "could this be function"?

```
# description
# Get the description for a pull request
# 
# description :: PullRequest -> String
def description(pull_request)
  if leankit_ticket_present?(pull_request)
    hotfix?(pull_request) ? "This is a hotfix!" :  slack_channel_topic
  else
    "None of the messages in the PR include LeanKit ID"

# status
# Get the status for a pull request
#
# status :: PullRequest -> String
def status(pull_request) 
  leankit_ticket_present?(pull_request) || hotfix?(pull_request) 
    ? build_status 
    : "error"
end
```